### PR TITLE
Update Marketo doc to include upsert leads toggle

### PR DESCRIPTION
**Pull request by ampcynthli**
# Amplitude Developer Docs PR

## Description

Update to reflect this option:
image

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs
@caseyamp

### CODE REVIEW

File: docs/data/destinations/marketo-cohort.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This integration combines Amplitude with Marketo's tools for in-app messaging, p
 ## Considerations
 
 - Email values are case-sensitive. If you use email address to map users between Amplitude and Marketo, the email address must be consistent between both systems.
-- Amplitude's cohort sync feature doesn't create new user profiles in Marketo. Instead, it flags whether an existing user on Marketo belongs to your Amplitude cohort.
+- If a user on Amplitude does not already exist in Marketo, the option to create the user can be toggled on or off with the "Upsert Leads" toggle on the cohort sync screen.
```
Comment (caseyamp) [inline docs/data/destinations/marketo-cohort.md:11]:
```suggestion
- Enable **Upsert Leads** on the cohort sync screen to create Amplitude users in Marketo if they don't already exist.
```

Comment (caseyamp) [reply docs/data/destinations/marketo-cohort.md:11]: @ampcynthli -- can you review my suggestion to make sure it works for you?

Comment (ampcynthli) [reply docs/data/destinations/marketo-cohort.md]: It looks great! Thank you

**Comment by ampcynthli** [comment]
Can be merged if all else looks good! Thanks for your help. I have another new doc on the way, and I'll try my best to read the guidelines.

**Review by caseyamp — APPROVED** [review]

**Pull request merged**